app/assets/stylesheets/hubstats/application.css

```diff
@@ -82,7 +82,8 @@
   margin: 0px 0px 15px 0px;
 }
 
-.pulls > .col, .comments > .col, .users > .col, .repos > .col, .deploys > .col {
+.pulls > .col, .comments > .col, .users > .col, .repos > .col, .deploys > .col
```

Comment:
`.deploys > .col` is duplicated 3 times and you are missing a comma to continue this line.